### PR TITLE
TOTP: second followup

### PR DESCRIPTION
**Pull request by xmo-odoo**
A few more fixes or improvements found since the "leftovers" were changed / fixed / ...

### CODE REVIEW

**Comment by xmo-odoo** [comment]
@robodoo rebase-ff

File: addons/mrp/models/stock_orderpoint.py [hunk 1]
```diff
@@ -24,7 +24,11 @@ def _get_replenishment_order_notification(self):
                 'tag': 'display_notification',
                 'params': {
                     'title': _('The following replenishment order has been generated'),
-                    'message': '<a href="#action=%d&id=%d&model=mrp.production" target="_blank">%s</a>' % (action.id, production.id, production.name),
+                    'message': '%s',
+                    'links': [{
+                        'label': production.name,
+                        'url': f'#action={action.id}&id={production.id}&model=mrp.production'
+                    }],
```
Comment (xmo-odoo) [inline addons/mrp/models/stock_orderpoint.py:31]: @amoyaux

Comment (amoyaux) [reply addons/mrp/models/stock_orderpoint.py:31]: Ok

**Comment by xmo-odoo** [comment]
FIXME: apparently the "styllistic" changes broke stuff (probably because of the reformatting of the secret which was somewhat expected)

File: doc/webservices/odoo.rst [hunk 2]
```diff
@@ -146,6 +146,50 @@ parameters
...
+Click :guilabel:`Generate Key`, then copy the key provided. **Store this key
+carefully**: it is equivalent to your password, and just like your password
+Odoo will not be able to retrieve or show the key again later on. If you lose
+this key, you will have to create a new one (and probably delete the one you
+lost).
+
+Once you have keys configured on your account, they will appear above the
+:guilabel:`New API Key` button, and you will be able to delete them:
+
+.. figure:: images/delete-key.png
```
[the remark this answers is not in the document]
Comment (xmo-odoo) [reply doc/webservices/odoo.rst:187]: Can even be a bit smaller with a recompressor and zopfli.

File: addons/auth_totp/views/templates.xml [hunk 3]
```diff
@@ -15,7 +15,7 @@
                     <div class="form-group">
                         <label for="totp_token">Authentication Code</label>
                         <input name="totp_token" class="form-control mb-2"
-                               autofocus="autofocus" required="required" placeholder="6-digit code"/>
+                               autofocus="autofocus" required="required" placeholder="123 456"/>
```
[the remark this answers is not in the document]
Comment (xmo-odoo) [reply addons/auth_totp/views/templates.xml]: @odony 
> PS: I had not noticed that in Community the layout is all broken:

Should be fixed, it seems to specifically be website which breaks this, if website is not installed *or* web_enterprise is installed it's fine, which is probably why I didn't notice it.

**Comment by xmo-odoo** [comment]
@odony resolved the comments, only question left is the documentation bits, and maybe the other questions @LNAodoo had for you on the task wrt various style changes and the like.

**Comment by xmo-odoo** [comment]
Retargeted to master because adds a new view -> ci/template fails. I guess I'll merge in master then backport to saas-13.5 instead.

**Comment by xmo-odoo** [comment]
@odony still need your take on @LNAodoo's graphical / style / ... and layout remarks, as I assume you had good reasons to change them the way you did and not <some other way>.

**Comment by xmo-odoo** [comment]
TODO: Fix commit message of "better issuer label": issuer label is now domain (with fallback to company)

**Comment by xmo-odoo** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**